src/main/java/GameOfLife/Cell.java

```diff
@@ -0,0 +1,52 @@
+package GameOfLife;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+class Cell {
+    private final int x;
+    private final int y;
+    private boolean alive;
+
+    public Cell(int x, int y) {
+        this.x = x;
+        this.y = y;
+        this.alive = Math.random() < 0.5;
```

Comment:
from design perspective this is the moment when you are loosing the control over the creation. such randomization could be moved to different class.